nixos/lib/test-driver/nixos_test_driver/nixos_test_driver.py

```diff
@@ -278,9 +290,16 @@ def create_startcommand(args: Dict[str, str]) -> str:
         return start_command
 
     def is_up(self) -> bool:
+        """Check whether the machine is up and available."""
         return self.booted and self.connected
 
     def log(self, msg: str) -> None:
+        """Log the given ``msg``.
```

Comment:
Does Sphinx also shows type annotations these days?

Reply:
Yes, it includes the type hints in the function declaration just like you now write them. It won't put them in the Parameters list though.